specifications/api/swagger.yaml

```diff
@@ -1179,6 +1225,21 @@ paths:
           schema: *ApiWalletPutData
       responses: *responsesPutWallet
 
+  /wallets/{walletId}/coin-selections/random:
+    post:
+      operationId: selectCoins
+      tags: ["Coin Selections"]
```

Comment:
**[remark]**

You'll also need to add this tags to the `x-tagGroups` list earlier in the document (otherwise it won't show up in the rendered doc).

Reply:
Done. See: https://github.com/input-output-hk/cardano-wallet/pull/1152/commits/4bf9856958e96ab6a0136b3854b360b4dfbcdf56.